### ISSUE TITLE
Domain Registration & Azure Resources Confirmation

### ISSUE THREAD

**Issue by mitchelsellers** [dotnet-foundation/projects#12]
The DNN Project has a domain name that we would like to get registered in a manner that isn't tied to any specific community member, and once that is done we need to get an Azure Web App created for hosting, with SSL Certificate for that particular domain.

I currently own the domain in question and can transfer wherever it needs to go.

Is this all stuff that we can get from the .NET Foundation?

The website will have moderate traffic, expected to be 20-50k requests a month to a WebApi project.

**Comment by mitchelsellers** [comment]
Additionally, another question/request is around email hosting.  Is there any Office365, or other solution available for projects?

**Comment by jongalloway** [comment]
This is a few issues mixed together, so we should probably split them out.

* Domain registration - Yes, we've got Azure DNS set up that makes it pretty easy to manage DNS, and you've already got an admin team set up in Azure so that'd be pretty straightforward
* Azure App Service - Also pretty straightforward, would just need specifics. I can create and put in your existing resource group so the admin team can manage
* SSL Cert - We can buy if needed, but as an MVP you should be able to get a free one under [DigiCert's MVP program](https://www.digicert.com/friends/msmvp/)
* E-mail hosting - We can set that up under our .NET Foundation O365 account. Several projects just have distribution lists set up, so info@projectname.com goes to the project leaders. If we need to set up full e-mail accounts we can do that, but it gets a little pricey so we try not to jump to that if it's not fully needed.

Based on that info, we should split this out to 4 issues if you want to go ahead with all of these so different people can help on individual items.

**Comment by mitchelsellers** [comment]
Perfect, closing this out and will revisit individual items
